### PR TITLE
Suppress error message when translation file doesn't exist

### PR DESCRIPTION
**Pull request by sharifelgamal**

### CODE REVIEW

File: pkg/minikube/translate/translate.go [hunk 1]
```diff
@@ -77,13 +77,13 @@ func DetermineLocale() {
 	translationFile := "pkg/minikube/translate/translations/" + preferredLanguage.String() + ".json"
 	t, err := ioutil.ReadFile(translationFile)
 	if err != nil {
-		glog.Errorf("Failed to load transalation file for %s: %s", preferredLanguage.String(), err)
+		glog.Infof("Failed to load translation file for %s: %s", preferredLanguage.String(), err)
```
Comment (medyagh) [inline pkg/minikube/translate/translate.go]: lets change the second %s to %v for err

File: pkg/minikube/translate/translate.go [hunk 2]
```diff
@@ -77,13 +77,13 @@ func DetermineLocale() {
...
 		return
 	}
 
 	err = json.Unmarshal(t, &Translations)
 	if err != nil {
-		glog.Errorf("Failed to populate translation map: %s", err)
+		glog.Infof("Failed to populate translation map: %s", err)
```
Comment (medyagh) [inline pkg/minikube/translate/translate.go]: %v for err here too

File: pkg/minikube/translate/translate.go (hunk 1 above)
Comment (sharifelgamal) [reply pkg/minikube/translate/translate.go]: Done.

File: pkg/minikube/translate/translate.go (hunk 2 above)
Comment (sharifelgamal) [reply pkg/minikube/translate/translate.go]: Done.

File: pkg/minikube/translate/translate.go [hunk 3]
```diff
@@ -77,13 +77,13 @@ func DetermineLocale() {
 	translationFile := "pkg/minikube/translate/translations/" + preferredLanguage.String() + ".json"
 	t, err := ioutil.ReadFile(translationFile)
 	if err != nil {
-		glog.Errorf("Failed to load transalation file for %s: %s", preferredLanguage.String(), err)
+		glog.Infof("Failed to load translation file for %s: %v", preferredLanguage.String(), err)
```
Comment (tstromberg) [inline pkg/minikube/translate/translate.go]: I may be incorrect, but I think the call to String() is likely unnecessary,  as %s should call it automatically.

Comment (sharifelgamal) [reply pkg/minikube/translate/translate.go]: You're probably right yeah.

**Pull request merged**